pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -158,22 +161,25 @@ public class ServiceConfiguration implements PulsarConfiguration {
     @FieldContext(
             category = CATEGORY_SERVER,
             required = false,
-            doc = "The metadata store URL for the configuration data. If empty, we fall back to use metadataStoreUrl"
+            doc = """
+                    The metadata store URL for the configuration data. If empty, we fall back to use metadataStoreUrl"""
```

Comment:
```suggestion
            doc = "The metadata store URL for the configuration data. If empty, we fall back to use metadataStoreUrl"
```

Revert. No need to change.